Documentation/gettingstarted/external-workloads.rst

```diff
@@ -45,189 +45,98 @@ Prerequisites
 * This guide assumes your external workload manages domain name
   resolution service via systemd.
 
+* So far this functionality is only tested with the vxlan tunneling
+  datapath mode (default for most installations).
+
```

Comment:
Initially, I tested this guide on the runtime VM which still has 4.9 kernel and thus doesn't support host reachable services which the Cilium agent started by the `install-external-workload.sh` script enables. Might be worth to point out in the prerequisites that external workloads thus require at least kernel 4.17 where BPF host reachable services are supported? Or do we not strictly needs host reachable services, in which case it might be good to remove `--enable-host-reachable-services` from the generated install script.

Reply:
Unfortunately ``--enable-host-reachable-services`` is a strict requirement. This is documented in the *Prerequisites* section of the GSG.

Reply:
Sorry, I somehow missed that during the review. It's completely fine as is.